### DISCUSSION TITLE
Shairport-Sync Multiroom.. No Audio on Ubuntu

### DISCUSSION THREAD

**Discussion by bibberp** [mikebrady/shairport-sync#1273] [Q&A] [answered]
Hello!
Like most, I'm thrilled with the new Airplay 2 support! I've followed Shairport-Sync for a few years and it's always been fantastic!
A couple bugs that I've found which may or may not be on the Shairport side:

1. I use Youtube music as my streaming service, sometimes when I select a new song or push the next button, I won't get any audio until I restart the song, then usually it starts playing. This never happened before
2. Sometimes the app will close while I'm using it only when I'm connected to Shairport-Sync. Could also be a YouTube music update

On to my issue:

Not sure if this should go to issues or stay in discussions..

I use the Shairport Sync container on a few computers, an Ubuntu, a Debian and a Raspberry.. for some reason I can't get sound out of the ubuntu computer anymore. The container seems like it's running fine, I tried repulling the container, the unstable and the stable one. When I connect to it it connects and acts like it's working so that makes me feel like it might be a sound card issue? Not sure where to start.

**Comment by charlesomer** [comment, marked as answer]
Hmm, are you able to play audio from something else on the Ubuntu machine?

**Reply by bibberp** [reply]
Good suggestion. I ended up reinstalling PA and ALSA, then I went into ALSA and unmuted everything and now it is working well! Thanks for your help!

**Reply by charlesomer** [reply]
No problem, glad you could get it working!
